### PR TITLE
document manually calling PUSH()

### PR DESCRIPTION
**Pull request by kmulvey**

### CODE REVIEW

**Comment by Johann-S** [comment]
In the next release (2.1.0) Ratchet has his own namespace so ``PUSH`` can be called like that too : 
``RATCHET.push(options)`` (https://github.com/twbs/ratchet/blob/master/js/push.js#L508)

I think it would be a good things to add this way to call ``PUSH`` on the documentation.

**Comment by kmulvey** [comment]
does the comment "Remove this line in the next major version" mean you will only be able to use it via the RATCHET namespace?

**Comment by Johann-S** [comment]
Yes but not on the next release 2.1.0 maybe on Ratchet 3.0

**Comment by kmulvey** [comment]
ok ill leave that out for now, i just write that you can use both as of 2.1.0

**Comment by XhmikosR** [comment]
@kmulvey: please squash the patches into one.

File: docs/components.html [hunk 1]
```diff
@@ -1261,6 +1261,13 @@ <h1 class="title">Push</h1>
 window.addEventListener('push', myFunction);
 {% endhighlight %}
 
+        <p class="component-description">Push.js can be triggered manually by using the PUSH(options) function. When passing options, url is reqired, container, transition and timeout are optional. As of version 2.1.0 Ratchet has his own namespace and can also be called with RATCHET.push(options).</p>
```
Comment (cvrebert) [inline docs/components.html]: * his => its
* "required" (note the "u")

**Pull request closed without merging**